package.json

```diff
@@ -35,13 +35,21 @@
     "init": "npm install && lerna bootstrap && npm run build",
     "link-all-packages": "lerna exec --ignore @leafygreen-ui/lib --ignore @leafygreen-ui/theme -- npm link ",
     "link": "./scripts/link.sh",
-    "lint": "eslint \"packages/**/*.js\"",
-    "prettier": "prettier --write \"**/*.js\"",
+    "lint": "npm run prettier:lint && npm run eslint",
+    "eslint": "eslint \"**/*.{js,ts,tsx}\"",
+    "prettier": "prettier --write \"**/*.{js,ts,tsx}\"",
+    "prettier:lint": "prettier --check \"**/*.{js,ts,tsx}\"",
```

Comment:
nit, let's align with the flag and have it be `prettier:check`. Also should probably have it be `--list-different` so we can see what files are not formatted correctly.

Reply:
Also should probably add `.md` as a file to use prettier on as well.